bigquery/go.mod

```diff
@@ -2,19 +2,22 @@ module cloud.google.com/go/bigquery
 
 go 1.15
 
+require (
+	github.com/googleapis/gax-go/v2 v2.2.0
+	google.golang.org/api v0.74.0
+	google.golang.org/genproto v0.0.0-20220401170504-314d38edb7de
+	google.golang.org/grpc v1.45.0
+	google.golang.org/protobuf v1.28.0
+)
+
 require (
 	cloud.google.com/go v0.100.2
 	cloud.google.com/go/datacatalog v1.3.0
 	cloud.google.com/go/iam v0.3.0
-	cloud.google.com/go/storage v1.21.0
+	cloud.google.com/go/storage v1.10.0
```

Comment:
Any idea why this dependency version was lowered to v1.10.0?

Reply:
That one's surprising.  I'll bump it up and tidy again.